pkg/openstacknet/funcs.go

```diff
@@ -85,3 +85,23 @@ func GetOpenStackNetWithLabel(r common.ReconcilerCommon, namespace string, label
 	}
 	return &osNetList.Items[0], nil
 }
+
+// GetOpenStackNetworkHostnameIPs - Return a map of hostname/IPs for the specified network selector
+func GetOpenStackNetworkHostnameIPs(r common.ReconcilerCommon, namespace string, labelSelector map[string]string) (map[string]string, error) {
+	ctlplaneIps := map[string]string{}
```

Comment:
would be good if this returns a map of a struct with role -> hostname, ctlplaneip . with this we can generate in the inventory also the groups that it is easier to register the different groups to different channels, like ceph nodes